source/client-side-encryption/tests/local.yml

```diff
@@ -1,56 +0,0 @@
-runOn:
```

Comment:
This slipped in originally, but ended up being renamed and hence no longer used.